RhythmTests/EasingTests.swift

```diff
@@ -0,0 +1,157 @@
+//
+//  EasingTests.swift
+//  Rhythm
+//
+//  Created by Brian Heim on 5/31/17.
+//
+//
+
+import XCTest
+@testable import Rhythm
+
+class EasingTests: XCTestCase {
+
+    //****** General errors *******//
```

Comment:
The idiomatic separator would be `// Mark: General Errors` which has a semantic used by editors and documentation generators.